src/test/resources/com/microsoft/jenkins/azuread/integrations/casc/ConfigAsCodeTest/config.xml

```diff
@@ -28,6 +28,8 @@
     <securityRealm class="com.microsoft.jenkins.azuread.AzureSecurityRealm">
         <clientid>clientId</clientid>
         <clientsecret>password</clientsecret>
+        <clientcertificate>password</clientcertificate>
```

Comment:
if the type is secret I don't see why this would be set?
```suggestion
        <clientcertificate></clientcertificate>
```

Reply:
Just this left and then ready to 🚢

Reply:
@timja applied, thanks.